users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -378,6 +396,69 @@ def run_scf_baseline_lr_reset():
     return report, ctc_nn_system
 
 
+def run_mel_baseline_lr_reset():
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/ctc/feat/"
+    _, nn_system = run_mel_baseline()
+    (
+        returnn_datasets,
+        rasr_loss_corpus_path,
+        rasr_loss_corpus_segments,
+        rasr_loss_lexicon_path,
+        dev_corpora,
+    ) = get_datasets()
+    returnn_args = {
+        "batch_size": 5000,
+        "rasr_binary_path": RASR_BINARY_PATH,
+        "rasr_loss_corpus_path": rasr_loss_corpus_path,
+        "rasr_loss_corpus_segments": rasr_loss_corpus_segments,
+        "rasr_loss_lexicon_path": rasr_loss_lexicon_path,
+        "datasets": returnn_datasets,
+        "extra_args": {
+            "accum_grad_multiple_step": 2,
+            "watch_memory": True,
+            "conv_pad_seq_len_to_power": 1.5,
+            "preload_from_files": {
+                "existing-model": {
+                    "filename": nn_system.train_jobs["conformer_bs2x5k_lgm80_baseline"].out_checkpoints[400],
+                    "init_for_train": True,
+                },
+            },
+        },
+        "conformer_type": "wei",
+        "specaug_old": {"max_feature": 15},
```

Comment:
```suggestion
        "specaug_old": {"max_feature": 8},
```
This deviates from the baseline. These things are really important! Please pay attention to the details.